clustering/jgroups/extension/src/main/java/org/jboss/as/clustering/jgroups/subsystem/JChannelFactoryBuilder.java

```diff
@@ -95,13 +96,23 @@ public JChannelFactoryBuilder(PathAddress address) {
         this.statisticsEnabled = StackResourceDefinition.Attribute.STATISTICS_ENABLED.resolveModelAttribute(context, model).asBoolean();
 
         Resource resource = context.readResourceFromRoot(this.address, false);
-        Optional<PathElement> transport = resource.getChildren(TransportResourceDefinition.WILDCARD_PATH.getKey()).stream().map(Resource.ResourceEntry::getPathElement).findFirst();
-        if (!transport.isPresent()) {
+        PathElement transport = null;
+        for (Resource.ResourceEntry resourceEntry : resource.getChildren(TransportResourceDefinition.WILDCARD_PATH.getKey())) {
+            transport = resourceEntry.getPathElement();
+            if (transport != null) break;
```

Comment:
Null check is unnecessary. A ResourceEntry always has a path. We can just break out of the loop here.